nominees/the-open-source-spatial-electrification-tool.json

```diff
@@ -29,9 +30,10 @@
   ],
   "organizations": [
     {
-      "name": "OnSSET",
-      "website": "https://www.onsset.org/",
-      "org_type": "owner"
+      "name": "KTH Energy Systems",
+      "website": "https://www.energy.kth.se/energy-systems/about-the-division-of-energy-systems-1.937036",
+      "org_type": "maintainer",
+      "contact_name": "KTH Royal Institute of Technology - division of Energy Systems"
```

Comment:
This appears to be an organization name, not a contact.

Reply:
Fixed in commit 22e3e92